src/mbgl/tile/geometry_tile_worker.cpp

```diff
@@ -319,6 +321,10 @@ void GeometryTileWorker::parse() {
         return;
     }
 
+#if MBGL_TILE_TIMING
```

Comment:
Instead of having the `#ifdefs` directly in the code, could we turn this into a macro, so that it's not as distracting from the rest of the code? So something like:

```
   MBGL_TIMING_START(tileTimer);

   ... logic you want to time...

   MBGL_TIMING_FINISH(tileTimer, [code that generates a timing log message]);
```

Reply:
Oh, I see @julianrex has already suggested something like along these lines. ➕